.github/workflows/chart-release.yaml

```diff
@@ -1,16 +1,7 @@
 name: Release Helm charts
 
 on:
-  workflow_run:
```

Comment:
won't this break packaging of the charts? `Update application version on dispatch` workflow depends on this